code/controllers/subsystem/processing/projectiles.dm

```diff
@@ -4,3 +4,20 @@ PROCESSING_SUBSYSTEM_DEF(projectiles)
 	stat_tag = "PP"
 	flags = SS_NO_INIT|SS_TICKER
 	var/global_max_tick_moves = 10
+	var/global_pixel_speed = 2
+	var/global_iterations_per_move = 16

```

Comment:
what this line do

Reply:
He already explained in the description. Increments projectiles by 2 pixels 16 times rather than 32 pixels once for greater collision accuracy.